integration-cli/docker_cli_images_test.go

```diff
@@ -288,3 +290,22 @@ func (s *DockerSuite) TestImagesFormatDefaultFormat(c *check.C) {
 	out, _ = dockerCmd(c, "--config", d, "images", "-q", "myimage")
 	c.Assert(out, checker.Equals, imageID+"\n", check.Commentf("Expected to print only the image id, got %v\n", out))
 }
+
+func (s *DockerSuite) TestImagePullWrongCreds(c *check.C) {
```

Comment:
This is the output from this PR code with the above fix (error is `no basic auth credentials`):
```
----------------------------------------------------------------------
FAIL: docker_cli_images_test.go:294: DockerRegistryAuthSuite.TestImagePullWrongCreds

docker_cli_images_test.go:310:
    c.Assert(out, checker.Contains, "401 Unauthorized")
... obtained string = "" +
...     "Using default tag: latest\n" +
...     "Error response from daemon: Get http://127.0.0.1:5000/v2/dockercli/busybox/manifests/latest: no basic auth credentials\n"
... substring string = "401 Unauthorized"
```